Project_Submission_braid.md

```diff
@@ -0,0 +1,56 @@
+
+# Project Name: Braid
+![image](https://user-images.githubusercontent.com/43886242/141393427-42887107-848c-4ede-a11e-24de8e357e0e.png)
+
+
+## Category 
+Dapp
+
+## Project Description
+Braid is trying to fix science publishing by reducing gatekeeping and increasing open access. 
+We want to be a place where scientists can upload their papers (to IPFS), review the work of their peers, and be compensated for both.
```

Comment:
Since this project is using web3.storage to upload papers, it will use both IPFS and Filecoin for the storage. IPFS as caching layer to provide fast store and retrieval, Filecoin as persistent layer for longterm and provable storage, that is who web3.storage works. 
So could you please update this part to reflect the usage of Filecoin as well? Not just IPFS. Thank you

Reply:
Updated text